compiler/src/iree/compiler/Dialect/LinalgExt/IR/TilingInterfaceImpl.cpp

```diff
@@ -73,7 +73,10 @@ SmallVector<utils::IteratorType> ScatterOp::getLoopIteratorTypes() {
   SmallVector<utils::IteratorType> iteratorTypes(getUpdateType().getRank(),
                                                  utils::IteratorType::parallel);
   if (!getUniqueIndices()) {
-    iteratorTypes[0] = utils::IteratorType::reduction;
+    int64_t numReductionLoops = getBatchRank();
```

Comment:
Nit: calling it `numReductionLoops` is confusing (though I understand why). Just call it `batchRank` and the code below makes it clear that batch ranks are made reductions.